### PR TITLE
Add test case for `defaultToString()` in `StringUtils`

### PR DESCRIPTION
**Pull request by skonda33**
Add test case for defaultToString method
---

I hereby agree to the terms of the [JUnit Contributor License Agreement](https://github.com/junit-team/junit-framework/blob/002a0052926ddee57cf90580fa49bc37e5a72427/CONTRIBUTING.md#junit-contributor-license-agreement).

---

### Definition of Done

- [x] There are no TODOs left in the code
- [x] Method [preconditions](https://docs.junit.org/snapshot/api/org.junit.platform.commons/org/junit/platform/commons/util/Preconditions.html) are checked and documented in the method's Javadoc
- [x] [Coding conventions](https://github.com/junit-team/junit-framework/blob/HEAD/CONTRIBUTING.md#coding-conventions) (e.g. for logging) have been followed
- [x] Change is covered by [automated tests](https://github.com/junit-team/junit-framework/blob/HEAD/CONTRIBUTING.md#tests) including corner cases, errors, and exception handling
- [x] Public API has [Javadoc](https://github.com/junit-team/junit-framework/blob/HEAD/CONTRIBUTING.md#javadoc) and [`@API` annotations](https://apiguardian-team.github.io/apiguardian/docs/current/api/org/apiguardian/api/API.html)
- [x] Change is documented in the [User Guide](https://docs.junit.org/snapshot/) and [Release Notes](https://docs.junit.org/snapshot/release-notes.html)

### CODE REVIEW

File: platform-tests/src/test/java/org/junit/platform/commons/util/StringUtilsTests.java [hunk 1]
```diff
@@ -128,6 +129,11 @@ void replaceWhitespaces() {
 		assertPreconditionViolationFor(() -> replaceWhitespaceCharacters("", null));
 	}
 
+	@Test
+	void defaultToStringChecks(){
+		assertEquals("null",defaultToString(null));
+	}
```
Comment (sbrannen) [inline platform-tests/src/test/java/org/junit/platform/commons/util/StringUtilsTests.java:135]: It doesn't make sense to call this method "...Checks" if there is only a single check/assertion.

To remedy that, you could introduce an additional assertion analogous to the `startsWith()` check in `nullSafeToStringForObjectWhoseToStringImplementationThrowsAnException()`.

----

In addition, the code is missing a space after the comma (",").

Did you run a full local build before submitting the PR?

**Review by sbrannen — DISMISSED** [review]

File: platform-tests/src/test/java/org/junit/platform/commons/util/StringUtilsTests.java (hunk 1 above)
Comment (skonda33) [reply platform-tests/src/test/java/org/junit/platform/commons/util/StringUtilsTests.java:135]: addressed review comments.

File: platform-tests/src/test/java/org/junit/platform/commons/util/StringUtilsTests.java [hunk 2]
```diff
@@ -128,6 +129,12 @@ void replaceWhitespaces() {
 		assertPreconditionViolationFor(() -> replaceWhitespaceCharacters("", null));
 	}
 
+	@Test
+	void defaultToStringReturnsNullForNullInput() {
+		assertEquals("null", defaultToString(null));
+
```
Comment (marcphilipp) [inline platform-tests/src/test/java/org/junit/platform/commons/util/StringUtilsTests.java]:
```suggestion
```

**Comment by marcphilipp** [comment]
Thanks, @skonda33! :+1:

**Review by marcphilipp — APPROVED** [review]
